web/src/components/review-form.jsx

```diff
@@ -128,13 +129,22 @@ export default class ReviewForm extends React.Component {
           </section>
         )) }
 
+        <section id="confirm-buttons" className="divCenter">
+
+        { this.state.pendingSentences ?
```

Comment:
can you move the indentation in two spaces as we're inside the `section` here? (applies for the lines below as well)

Reply:
Done!